### DISCUSSION TITLE
how do I make my character to ride a vehicle?

### DISCUSSION THREAD

**Discussion by ckdlskgkdrhd** [naverz/zepeto-studio-kor#1222] [5. Q&A]
I'm guessing Zepeto Studio itself doesn't support this and I'd have to make a script myself
but how would I go about disabling the zepeto controller input to read the input from car controller.
or am I thinking this wrong, if so I could certainly use some tips.
would be really nice if there was a sample guide for this.

**Comment by zlmrzlo** [comment]
zepeto controller input을 비활성화하는 방법은 hierarchy에 있는 zepetoPlayers에 있는 UIController를 비활성화하는 것입니다. 런타임에 접근해서 비활성화시키면 됩니다.
